packages/core/src/validation.ts

```diff
@@ -57,6 +57,7 @@ const wallet = Joi.object({
   label: Joi.string(),
   icon: Joi.string(),
   provider: unknownObject,
+  instance: unknownObject.optional(),
```

Comment:
```suggestion
  instance: unknownObject,
```
Joi validations default to optional so this can be removed.